scripts.d/.plan-ng-dev

```diff
@@ -40,13 +41,20 @@ s-cmake/native
         s-zeromq:skip
         s-melissa:skip
         s-persalys:skip
+        s-zcracks:skip
 ]
 
 [ = deb,9
         s-salome-ng:s9-dev
         s-salome-tar:#master:DB09
         s-cmake:skip
         s-zeromq:skip
+        s-cgns:skip
```

Comment:
All those seems not related to zcracks module?

Reply:
Thanks for removing the debug, but what about all those skipped plug-ins? This can't be related to s-zcracks as it does not bring a single dependency, right? So to me all this is superfluous. Please clarify. TIA.

Reply:
Yes you're right It doesn't concern Zcracks. I can remove it to push in another branch.